repoqa/provider/google.py

```diff
@@ -18,9 +18,10 @@ def __init__(self, model):
         self.client = genai.GenerativeModel(model)
 
     def generate_reply(
-        self, question, n=1, max_tokens=1024, temperature=0, system_msg=None
+        self, question, n=1, max_tokens=1024, temperature=0, system_msg=None, stop=None
     ) -> List[str]:
         assert temperature != 0 or n == 1, "n must be 1 when temperature is 0"
+        assert stop is None, "stop is not supported in Google provider"
```

Comment:
gemini supports `stop_sequences` as well but it's in `GenerationConfig`. so it need to edit the impl of `make_auto_request`